src/lib/i18next.js

```diff
@@ -0,0 +1,44 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import i18next from 'i18next';
+
+import TRANSLATIONS_DE from './translations/de/translations.json';
+import TRANSLATIONS_EN from './translations/en/translations.json';
+import TRANSLATIONS_TR from './translations/tr/translations.json';
+
+import LanguageDetector from 'i18next-browser-languagedetector';
+
+i18next
+  // detect user language
+  .use(LanguageDetector)
+  .init({
```

Comment:
Are the translations here part of the production bundle? There is no request happening once the js is loaded right?

Reply:
right - they are bundled with the help of rollup, after production bundle they are found in the `/dist/cjs/index.js` & `/dist/esm/index.js`.